pkg/cloud/provider/csvprovider.go

```diff
@@ -308,10 +308,10 @@ func NodeValueFromMapField(m string, n *v1.Node, useRegion bool) string {
 		if mf[1] == "name" {
 			return toReturn + n.Name
 		} else if mf[1] == "labels" {
-			lkey := strings.Join(mf[2:], "")
+			lkey := strings.Join(mf[2:len(mf)], ".")
```

Comment:
The `len(mf)` shouldn't be necessary.

Reply:
Some kind of merge conflict happened here where the "." below got ommitted when merging from develop which I think removed the len(mf). Will remove.